app/assets/javascripts/item_selector/item_selector_incrementor.js

```diff
@@ -0,0 +1,51 @@
+var itemSelectorIncrementor = (function() {
+  return $.extend(itemSelector, {
+    init: function() {
+      var _this = this;
+      $(document).on('ready page:load', function(){
+        _this.setDefaultItemCounter();
+        _this.addIncrementBehavior();
+      });
+    },
+
+    addIncrementBehavior: function() {
+      var _this = this;
+      _this.checkboxes().each(function() {
+        $(this).on('item-selector:selected', function() {
+          _this.increaseSelectedItemCount();
+        });
+
+        $(this).on('item-selector:deselected', function() {
+          _this.decreaseSelectedItemCount();
+        });
+      });
+    },
+
+    increaseSelectedItemCount: function() {
+      this.selectedItemCounter().text(
+        parseInt(this.selectedItemCounter().text(), 10) + 1
```

Comment:
I was not.  It didn't seem like it was correctly set in order that it would have needed to be.  (maybe would have needed a promise or something to work like that, but that seemed like a lot of overhead).